<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dama</class>
 <widget class="QMainWindow" name="Dama">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dama2013</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource1.qrc">
    <normaloff>:/img/stone/black_dame.png</normaloff>:/img/stone/black_dame.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>638</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionLoad"/>
    <addaction name="actionLoadReplay"/>
    <addaction name="actionSave"/>
    <addaction name="actionAbort"/>
    <addaction name="separator"/>
    <addaction name="actionHint"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_game"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Main toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionLoad"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionHint"/>
  </widget>
  <widget class="QToolBar" name="replayToolBar">
   <property name="windowTitle">
    <string>Replay toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBack_to_origin"/>
   <addaction name="actionPlay_auto"/>
   <addaction name="actionPause"/>
   <addaction name="actionBack"/>
   <addaction name="actionNext"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resource1.qrc">
     <normaloff>:/img/new.ico</normaloff>:/img/new.ico</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="resource1.qrc">
     <normaloff>:/img/open.ico</normaloff>:/img/open.ico</iconset>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resource1.qrc">
     <normaloff>:/img/exit.ico</normaloff>:/img/exit.ico</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="resource1.qrc">
     <normaloff>:/img/settings.ico</normaloff>:/img/settings.ico</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionAbout_game">
   <property name="icon">
    <iconset resource="resource1.qrc">
     <normaloff>:/img/about.ico</normaloff>:/img/about.ico</iconset>
   </property>
   <property name="text">
    <string>About game</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resource1.qrc">
     <normaloff>:/img/save.ico</normaloff>:/img/save.ico</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionHint">
   <property name="icon">
    <iconset resource="resource1.qrc">
     <normaloff>:/img/hint.ico</normaloff>:/img/hint.ico</iconset>
   </property>
   <property name="text">
    <string>Hint</string>
   </property>
  </action>
  <action name="actionAbort">
   <property name="icon">
    <iconset resource="resource1.qrc">
     <normaloff>:/img/close.ico</normaloff>:/img/close.ico</iconset>
   </property>
   <property name="text">
    <string>Abort</string>
   </property>
  </action>
  <action name="actionPlay_auto">
   <property name="icon">
    <iconset resource="resource1.qrc">
     <normaloff>:/img/play.ico</normaloff>:/img/play.ico</iconset>
   </property>
   <property name="text">
    <string>Play auto</string>
   </property>
   <property name="toolTip">
    <string>Play automatic</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="resource1.qrc">
     <normaloff>:/img/pause.ico</normaloff>:/img/pause.ico</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="toolTip">
    <string>Pause replay</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="icon">
    <iconset resource="resource1.qrc">
     <normaloff>:/img/seek-back.ico</normaloff>:/img/seek-back.ico</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="toolTip">
    <string>Back last draught</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="resource1.qrc">
     <normaloff>:/img/seek-fwd.ico</normaloff>:/img/seek-fwd.ico</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="toolTip">
    <string>Next draught</string>
   </property>
  </action>
  <action name="actionLoadReplay">
   <property name="icon">
    <iconset resource="resource1.qrc">
     <normaloff>:/img/open.ico</normaloff>:/img/open.ico</iconset>
   </property>
   <property name="text">
    <string>Load replay</string>
   </property>
   <property name="toolTip">
    <string>Load replay</string>
   </property>
  </action>
  <action name="actionBack_to_origin">
   <property name="icon">
    <iconset resource="resource1.qrc">
     <normaloff>:/img/skip-back.ico</normaloff>:/img/skip-back.ico</iconset>
   </property>
   <property name="text">
    <string>Back to origin</string>
   </property>
   <property name="toolTip">
    <string>Back to origin</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource1.qrc"/>
 </resources>
 <connections/>
</ui>
